wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -73,8 +74,8 @@ namespace :wagn do
 
   desc "migrate structure and cards"
   task :migrate =>:environment do
-    ENV['SCHEMA'] ||= "#{Wagn.root}/db/schema.rb"
-    
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
```

Comment:
I think the reason for this chunk of code is that schema migrations tend to try to update the schema.rb and we used this to hack around that problem somehow.

In general, my priority is trying NOT to put stuff in decks (schemas, routes, etc) that we may want to update.  I'm fine with making it so that folks can override default stuff in their decks; whenever you override behavior, you're willfully taking on a maintenance chore.

That said, the decko-rails case is almost certainly going to involve more overrides on average, because it's trying to squeeze wagn into a different app. So yeah, we need to be prepared for that.